src/Umbraco.Core/Configuration/Models/SecuritySettings.cs

```diff
@@ -116,4 +117,6 @@ public class SecuritySettings
     /// </summary>
     [DefaultValue(StaticAuthorizeCallbackPathName)]
     public string AuthorizeCallbackPathName { get; set; } = StaticAuthorizeCallbackPathName;
+
+    public string AuthorizeCallbackErrorPathName { get; set; } = StaticAuthorizeCallbackErrorPathName;
```

Comment:
I think you need to provide a default value here:

```suggestion
    [DefaultValue(StaticAuthorizeCallbackErrorPathName)]
    public string AuthorizeCallbackErrorPathName { get; set; } = StaticAuthorizeCallbackErrorPathName;
```